internal/parser/scanner/scanner.go

```diff
@@ -112,8 +111,10 @@ func (s *scanner) Next() token.Token {
 		return scanUKeyword(s)
 	case 'V':
 		return scanVKeyword(s)
-	// case ' ':
-	// 	return scanSpace(s)
+	case 'W':
+		return scanWKeyword(s)
+	case ' ':
```

Comment:
When I call `scanner.Next()`, and the next rune is `' '`, then I get nil returned. Please handle whitespaces between tokens correctly